uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeavePeriodEntitlement.php

```diff
@@ -591,6 +591,9 @@ public static function getRemainder($params){
    * whether the expired parameter is true or false
    *
    * @param array $params
+   *   [
+   *    'entitlement_id' => 1
```

Comment:
This description really doesn't help much. There are other values accepted here.

The purpose of the description is so that people can read it and understand what the function does and how to use it without having to read the code. Please keep that in mind when writing docblocks.

For this specific case, I think it would be better for the description to be something like this:

```php
@param array $param
  The param array passed to the LeavePeriodEntitlement.getBreakdown API endpoint.
  The supported values are:
  - entitlement_id: The id for a LeavePeriodEntitlement
  - contact_id: The id for a Contact
  - period_id: The id for a Absence Period
  - expired: A boolean flag. When it's true, only expired records will be returned. Otherwise, only non-expired 
```